### PR TITLE
Fixing the installer script

### PR DESCRIPTION
**Pull request by bigbitbus**
This PR modifies the end of the install script to tell the user if ln failed, in which case it suggests a couple of ideas on how to proceed.

### CODE REVIEW

**Review by juandiegopalomino — APPROVED** [review]

File: static/install.sh [hunk 1]
```diff
@@ -111,16 +111,15 @@ echo "Installing..."
...
-  RUNPATH=~/.opta/opta
-  echo "To add opta to your path, please add" 'export PATH=$PATH:'"$RUNPATH" 'to your terminal profile.'
+  echo "Please symlink the opta binary to one of your path directories; for example using 'sudo ln -fs ~/.opta/opta /usr/local/bin/opta'"
+  echo "Alternatively, you could add the .opta installation directory to your path like so"
+  echo "export PATH=$PATH:"$RUNPATH""
+  echo "to your terminal profile."
 fi
-
-echo "Successfully installed! Now you can run it via invoking $RUNPATH"
+echo "Successfully installed opta at location ~/.opta"
```
Comment (legalosLOTR) [inline static/install.sh:124]: This message should be different for both the if cases no?

Comment (bigbitbus) [reply static/install.sh:124]: The program is always installed at ~/.opta
The L117 will clarify to the user how to invoke it incase the symlink worked.

Comment (legalosLOTR) [reply static/install.sh:124]: But users mostly just read the last line! So it would be better for the last line to be "how to invoke".

Comment (bigbitbus) [reply static/install.sh:124]: Ok, I'll remove the last line. Then the user either sees 

```
 echo "Opta symlinked to /usr/local/bin/opta; You can now type 'opta' in the terminal to run it."
```

OR

```
  echo "Please symlink the opta binary to one of your path directories; for example using 'sudo ln -fs ~/.opta/opta /usr/local/bin/opta'"
  echo "Alternatively, you could add the .opta installation directory to your path like so"
  echo "export PATH=$PATH:"$RUNPATH""
  echo "to your terminal profile."
```

**Review by NitinAgg — APPROVED** [review]

**Pull request merged**